From ecc5dbfe96c7e438f9c9c7894971cc4c371bae35 Mon Sep 17 00:00:00 2001
From: Xilin Wu <sophon@radxa.com>
Date: Sat, 9 May 2026 14:19:14 +0800
Subject: [PATCH 179/268] arm64: configs: Update radxa_qcom_7_0_defconfig

Disable ladder and menu cpuidle governors, and use teo by default to
achieve stable 2.5 Gbps Ethernet on Radxa Dragon Q8B.

Signed-off-by: Xilin Wu <sophon@radxa.com>
---
 arch/arm64/configs/radxa_qcom_7_0_defconfig | 2 --
 1 file changed, 2 deletions(-)

--- a/arch/arm64/configs/radxa_qcom_7_0_defconfig
+++ b/arch/arm64/configs/radxa_qcom_7_0_defconfig
@@ -73,8 +73,6 @@ CONFIG_PM_QOS_CPU_SYSTEM_WAKEUP=y
 CONFIG_PM_DEBUG=y
 CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
 CONFIG_ENERGY_MODEL=y
-CONFIG_CPU_IDLE_GOV_LADDER=y
-CONFIG_CPU_IDLE_GOV_MENU=y
 CONFIG_CPU_IDLE_GOV_TEO=y
 CONFIG_ARM_PSCI_CPUIDLE=y
 CONFIG_CPU_FREQ=y
